src/c++/library/common.cc

```diff
@@ -182,6 +182,15 @@ InferInput::AppendFromString(const std::vector<std::string>& input)
   return AppendRaw(reinterpret_cast<const uint8_t*>(&sbuf[0]), sbuf.size());
 }
 
+Error
+InferInput::RawData(const uint8_t** buf, size_t* byte_size)
+{
+  // TODO - handle multi-batch case
```

Comment:
let's file a ticket and tag it here instead of unlabeled TODOs

Reply:
Done